### PR TITLE
Skeleton

### PR DESCRIPTION
**Pull request by shiranmeir**
Skeleton from ant design : Text, Input, Avatar and Button.

Can see examples in playground .

### CODE REVIEW

File: src/components/common/WMSkeleton/WMSkeletonButton.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  className?: string;
+}
+
+export default function WMSkeletonButton({
+  className,
+  ...otherProps
+}: IWMSkeletonButton): ReactElement {
+  return (
+    <Skeleton.Button
+      className={cc([classes['wm-skeleton'], classes['avatar'], className])}
```
Comment (RonnyO) [inline src/components/common/WMSkeleton/WMSkeletonButton.tsx:19]: You mean `classes['button']`

File: src/components/Screen/Playground/components/Skeleton.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      Avatar - square
+      <WMSkeletonAvatar active size="default" shape="square" />
+      <Divider />
+      Button - round
+      <WMSkeletonButton active size="default" shape="round" />
+      Button - small circle
+      <WMSkeletonButton active size="small" shape="circle" />
+    </>
+  );
+}
```
Comment (RonnyO) [inline src/components/Screen/Playground/components/Skeleton.tsx:29]: Looking good!

File: src/components/common/WMSkeleton/style.module.scss [hunk 3]
```diff
@@ -1,5 +1,5 @@
-.wm-skeleton {
-  &:global(.ant-skeleton) {
-    width: auto;
-  }
-}
+// .wm-skeleton {
+//   &:global(.ant-skeleton) {
+//     width: auto;
+//   }
+// }
```
Comment (RonnyO) [inline src/components/common/WMSkeleton/style.module.scss:5]: Do you want to keep it?

**Comment by RonnyO** [comment]
@dvira please review

File: src/components/common/WMSkeleton/WMSkeletonButton.tsx (hunk 1 above)
Comment (shiranmeir) [reply src/components/common/WMSkeleton/WMSkeletonButton.tsx:19]: Yes!

File: src/components/common/WMSkeleton/style.module.scss (hunk 3 above)
Comment (shiranmeir) [reply src/components/common/WMSkeleton/style.module.scss:5]: I kept it because i thought i will use it in the future, 
deleted it for now anyway..

**Pull request merged**